### DISCUSSION TITLE
.Net Maui app Beyond Hello World

### DISCUSSION THREAD

**Discussion by manishbisen** [dotnet/maui#12249] [Q&A]
In twitter,youtube etc I see people mostly demonstrate on maui hello world app.Can I see people demonstrate on real world complex maui apps instead of hello world apps ?Or it is not capable to create complex apps ?

**Comment by ArthurGrot** [comment]
You can look at the [.NET Podcast](https://github.com/microsoft/dotnet-podcasts) reference app. The mobile part is build with MAUI.
